### PR TITLE
v1.12.1 template_basic_plugin - VERSION RELEASE - standardize install

### PR DESCRIPTION
**Pull request by mindyls**
# Related Issues
required to close NRLMMD-GEOIPS/geoips#529

# Reviewer Instructions
* Please confirm this PR is set up to merge from v1.12.2-release branch into main branch
* Review "Files changed" tab
* Confirm appropriate testing was completed for these updates (you may perform tests yourself, or review output included from another reviewer/assignee)

# Summary
Changes for version 1.12.1, uploaded with
1.12.2 update on repo template_basic_plugin.
See release note updates in 'Files changed' tab

See NRLMMD-GEOIPS/geoips#529 for additional repositories included in this update.

# Testing Instructions
See NRLMMD-GEOIPS/geoips#529 for testing instructions.

# Output
See NRLMMD-GEOIPS/geoips#529 for test output.

# Post Merge Steps
Once this PR has been merged, v1.12.1
will be tagged/released on the template_basic_plugin main branch.

### CODE REVIEW

**Review by christopher-selman — APPROVED** [review]

**Review by jsolbrig — APPROVED** [review]

File: my_package/_version.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+else:
+    VERSION_TUPLE = object
+
+version: str
+__version__: str
+__version_tuple__: VERSION_TUPLE
+version_tuple: VERSION_TUPLE
+
+__version__ = version = '1.11.7'
+__version_tuple__ = version_tuple = (1, 11, 7)
```
Comment (mindyls) [inline my_package/_version.py:16]: I think this is the wrong _version.py

Comment (mindyls) [reply my_package/_version.py]: I fixed it

**Comment by mindyls** [comment]
image

I am tempted to just get this one merged, and resolve the CI errors separately (currently taking issue with the fact the package is named "my_package" and the repo is named "template_basic_plugin").

I would like to get the layout / installation of the template_basic_plugin package finalized, so we can at least start testing.

**Comment by mindyls** [comment]
I am merging this because I am trying to test the tutorial, and I would like a clean copy from template_basic_plugin.

**Review by cpcamacho — APPROVED** [review]

**Review by jsolbrig — APPROVED** [review]

**Comment by mindyls** [comment]
Merge without checks passing, because we need to get the latest release notes updated in a new PR.

**Pull request merged**